### ISSUE TITLE
compass-twitter-bootstrap vs. sass-twitter-bootstrap

### ISSUE THREAD

**Issue by sindresorhus** [vwall/compass-twitter-bootstrap#86]
How does this compare with [sass-twitter-bootstrap](https://github.com/jlong/sass-twitter-bootstrap)?

The files look identical:
1. https://github.com/vwall/compass-twitter-bootstrap/blob/master/stylesheets/compass_twitter_bootstrap/_accordion.scss
2. https://github.com/jlong/sass-twitter-bootstrap/blob/master/lib/_accordion.scss

Is it just that this one can be installed as a Compass bundle?

Wouldn't it then be better to combine the efforts?

// @jlong

**Comment by apsylone** [comment]
+1

**Comment by rizkysyazuli** [comment]
yep.. the [sass-twitter-bootstrap](https://github.com/jlong/sass-twitter-bootstrap) is not a compass plugin.

that's why i came here actually. i'm looking for a good compass port. i found two project that seems to be most active. this one, [compass-twitter-bootstrap](https://github.com/vwall/compass-twitter-bootstrap) and the [bootstrap-sass](https://github.com/thomas-mcdonald/bootstrap-sass) project.

can @vwall explain the difference between these two ports? thx..

**Comment by vwall** [comment]
There was only one other gem when I created this and it was behind on the updates.  Ironic huh...

**Comment by burisu** [comment]
One good difference for me is that compass-twitter-bootstrap doesn't override compass mixins like `text-shadow` or `box-shadow`. So, there is no need to that kind of code: https://github.com/thomas-mcdonald/bootstrap-sass#passing-multiple-values-to-mixins
